mappings/net/minecraft/server/network/ServerPlayNetworkHandler.mapping

```diff
@@ -53,3 +53,4 @@ CLASS net/minecraft/class_3244 net/minecraft/server/network/ServerPlayNetworkHan
 	METHOD method_14372 syncWithPlayerPosition ()V
 	METHOD method_18784 tick ()V
 	METHOD method_19507 isServerOwner ()Z
+	METHOD method_20630 isPlayerCollidingWithBlocks (Lnet/minecraft/class_4538;)Z
```

Comment:
Shouldn't this be reversed since it returns `worldView.doesNotCollide(...)`?

Reply:
oh yes

Reply:
corrected